x/poolmanager/keeper_test.go

```diff
@@ -82,8 +80,6 @@ func (s *KeeperTestSuite) TestInitGenesis() {
 }
 
 func (s *KeeperTestSuite) TestExportGenesis() {
-	s.Setup()
```

Comment:
Doesn't s.Setup() ensure we are running tests in a clean(new) env?

Reply:
The suite concept guarantees that `SetupTest` is called at the beginning of every test!